.changeset/beige-plants-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'openzeppelin-solidity': patch
+---
+
+`ERC721Wrapper`: Restrict custom `data` encoded wrapping to trusted `this`
```

Comment:
The changeset is not needed because the feature was never released. The user-facing change in the next release will be simply the addition of ERC721Wrapper.